modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/LumenServerCodegen.java

```diff
@@ -154,7 +154,7 @@ public String escapeReservedWord(String name) {
      * instantiated
      */
     public String modelFileFolder() {
-        return outputFolder + "/" + sourceFolder + "/" + modelPackage().replace('.', File.separatorChar);
+        return outputFolder + "/" + modelPackage().replace('.', File.separatorChar);
```

Comment:
I think you need to replace `/` with `File.separatorChar` for `(outputFolder + "/" + modelPackage())`